packages/EasyCodingStandard/src/DependencyInjection/EasyCodingStandardKernel.php

```diff
@@ -102,6 +103,7 @@ protected function getContainerLoader(ContainerInterface $container): Delegating
         return new DelegatingLoader(new LoaderResolver([
             new GlobFileLoader($container, $fileLocator),
             new CheckerTolerantYamlFileLoader($container, $fileLocator),
+            new PhpFileLoader($container, $fileLocator),
         ]));
     }
 }
```

Comment:
Could you decouple that to factory to prevent missed place like in the issue?

E.g. `Symplify\EasyCodingStandard\DependencyInjection\Loader\LoaderFactory`

Reply:
Ok, and should it be reused in the kernel as well?